style/cart.css

```diff
@@ -0,0 +1,137 @@
+:root{
+    --primary-color:black;
+    --secondary-color:tan;
+    --active-color:aqua;
+}
+body{
+    background-color: tan;
+    padding: 0%;
+    border: 0%;
+    margin: auto;
+}
+.navigation ul a{
+    
+    text-decoration: none;
+    letter-spacing: 0.2rem;
+    color:var(--primary-color);
+}
+.navigation ul a:hover{
+    color:var(--active-color);
+}
+
+.nav2{
+    width:100%;
+    height: 1rem;
+    line-height: 1rem;
+    justify-content:space-evenly;
+    align-items:center;
+    box-shadow: none;
+    text-align: center;
+}
+.nav2 ul{
+    list-style: none;
+    /* text-align: center; */
+}
+.nav2 ul li{
+    display: inline-block;
+    width: 10rem;
+    position: relative;
+    z-index: 90;
+}
+.nav2 ul li a{
+    text-decoration: none;
+    color:var(--secondary-color);
+    display: block;
+}
+.nav2 ul li a:hover{
+    color: var(--active-color);    
+}
+.nav2 ul ul{
+    position:absolute;
+    left: 0;
+    top: 1rem;
+    display: none;;
+}
+.nav2 ul ul li{
+    background-color: tan;
+    border-bottom: 0.1rem solid white;
+    color: var(--primary-color);
+}
+.nav2 ul ul li a{
+    border-bottom: 0.1rem solid white;
+    list-style: var(--active-color);
+    background-color: var(--secondary-color);
+    color: var(--primary-color);
+}
+.nav2 ul li:hover >ul {
+    display: block;
+}
+.nav2 ul ul li:hover{
+    background-color: var(--secondary-color);
+    color: var(--primary-color);
+}
+.nav2 ul ul li a:hover{
+    background-color: var(--active-color);
+    color: var(--primary-color);
+}
+
+.cart-container{
+    display: flex;
+    justify-content: space-around;
+    padding: 1rem;
+    flex-wrap: wrap;
+}
+.item-container{
+    display: flex;
+    flex-direction: column;
+}
+.btn.btn_Success{
+    background-color:var(--active-color);
+    color: var(--primary-color);
+}
+.btn.btn_Success:hover{
+    background-color:#1e9c1e;
+    color: white;
+}
+
+
+
+
+footer{
+    background-color: #172b4df3;
+    width: 100%;
+    height: 15rem;
+    color: var(--secondary-color);
+    font-size: medium;
+    display:flex;
+    align-items:center;
+    justify-content: center;
+    text-align: center;
+}
+
+footer{
+    background-color: #172b4df3;
+    width: 100%;
+    height: 15rem;
+    color: var(--secondary-color);
+    font-size: medium;
+    display:flex;
+    align-items:center;
+    justify-content: center;
+    text-align: center;
+}
```

Comment:
I think by mistake you have specified footer CSS twice

Reply:
removed footer